test/test-viz.cpp

```diff
@@ -0,0 +1,311 @@
+#include <iostream>
+#include <vector>
+#include <string>
+#include <tuple>
+#include <variant>
+
+#include <lorina/bench.hpp>
+
+#define SDL_MAIN_HANDLED
+
+#include <SDL.h>
+
+enum SPACING {
+    HSPACING = 2,
+    VSPACING = 2
+};
+
+enum ERROR_CODES {
+    EMPTY_FILENAME = -1,
+    PARSER_FAILURE = -2,
+    SDL_INIT_FAILURE = -3
```

Comment:
There is a convention to prefix related enumeration names.
If actual enum values are not important, we can omit them.
```suggestion
    ERROR_EMPTY_FILENAME = 1, // non-zero to indicate a failure, subsequent values are increments
    ERROR_PARSER_FAILURE,
    ERROR_SDL_INIT_FAILURE
```